service/src/test/resources/application-integrationtest.properties

```diff
@@ -0,0 +1,36 @@
+server.error.include-message=always
+###### RDBS via JPA #####
+spring.jpa.orm=orm.xml
+#spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.open-in-view=true
+server.port=8090
```

Comment:
Is this needed? I assume that random ports are used in the tests and setting a port in the config would not be needed.

Reply:
You're right, random pprts are used. I removed the port.